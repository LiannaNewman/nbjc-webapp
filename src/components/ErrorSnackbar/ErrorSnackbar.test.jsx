// import React from 'react';
// import { render } from '@testing-library/react';

// import ErrorSnackbar from './ErrorSnackbar';

// test('ErrorSnackbar', () => {

// });
