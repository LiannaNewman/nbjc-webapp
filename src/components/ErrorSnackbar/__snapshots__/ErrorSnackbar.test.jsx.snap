// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorSnackbar should render with default props 1`] = `
<DocumentFragment>
  <div
    class="MuiSnackbar-root MuiSnackbar-anchorOriginBottomCenter"
  >
    <div
      class="MuiPaper-root MuiAlert-root MuiAlert-standardError ErrorSnackbar-alert-1 MuiPaper-elevation0"
      direction="up"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiAlert-message ErrorSnackbar-alertMessage-2"
      >
        <p
          class="MuiTypography-root MuiTypography-body2"
        >
          Test body message
        </p>
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiButton-root MuiButton-text ErrorSnackbar-supportButton-4 MuiButton-disableElevation"
          href="/support"
          tabindex="0"
        >
          <span
            class="MuiButton-label"
          >
            Support
          </span>
        </a>
      </div>
      <div
        class="MuiAlert-action"
      >
        <button
          aria-label="Close"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeSmall"
          tabindex="0"
          title="Close"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`ErrorSnackbar should render without support link 1`] = `
<DocumentFragment>
  <div
    class="MuiSnackbar-root MuiSnackbar-anchorOriginBottomCenter"
  >
    <div
      class="MuiPaper-root MuiAlert-root MuiAlert-standardError ErrorSnackbar-alert-5 MuiPaper-elevation0"
      direction="up"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiAlert-message ErrorSnackbar-alertMessage-6"
      >
        <p
          class="MuiTypography-root MuiTypography-body2"
        >
          Test body message
        </p>
      </div>
      <div
        class="MuiAlert-action"
      >
        <button
          aria-label="Close"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeSmall"
          tabindex="0"
          title="Close"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
